code/datums/voice_of_god_command.dm

```diff
@@ -159,7 +159,7 @@ GLOBAL_LIST_INIT(voice_of_god_commands, init_voice_of_god_commands())
 
 /datum/voice_of_god_command/sleeping/execute(list/listeners, mob/living/user, power_multiplier = 1, message)
 	for(var/mob/living/carbon/target as anything in listeners)
-		target.Sleeping(40 * power_multiplier)
+		target.Sleeping(20 * power_multiplier)
```

Comment:
```suggestion
		target.Sleeping(30 * power_multiplier)
```
if the duration is too short, there's not much of a reason to use this instead of a stun command, since they'll wake up before they can finish typing whatever they wanted to say

Reply:
sleep is already overpowered as you literally put somone asleep but you can still use comms when they use stun or the other ones,sleep completely stops you from fighting back which is why i think it should be 2 seconds only